### PR TITLE
H2oamidoc pratap

### PR DESCRIPTION
**Pull request by pratapram**
Adding small documentation for a publishing a new AMI on AWS.

### CODE REVIEW

**Comment by michalkurka** [comment]
@tomkraljevic please review

File: h2o-docs/src/product/productionizing.rst [hunk 1]
```diff
@@ -210,6 +210,25 @@ Resource     Location
 Web sites    http://www.h2o.ai/steam/
 =========    ==================================================================================================
 
+Scoring Server on AWS
+~~~~~~~~~~~~~~~~~~~~~
+You can deploy a RESTful server on AWS, using the marketplace AMI (H2O Inference server - Hourly). Notice that this is a paid AMI. Transfer the
```
Comment (mmalohlava) [inline h2o-docs/src/product/productionizing.rst]: Which H2O inference server?

Comment (angela0xdata) [reply h2o-docs/src/product/productionizing.rst]: I couldn't find "h2o inference server" in the Amazon marketplace.

**Comment by pratapram** [comment]
> I would say the way one would review this would be to try to follow the directions to get it to work. But looking at the changes, there isn't enough information there for me to know how to do that.

I am creating a new AWS AMI to help with deploying h2o3 mojos. This is a simple rest server. There is a catch-22 with this instructions. You cannot follow the instructions unless the AMI is published. AWS will not publish this AMI unless they see this instruction.

So, please publish this instruction or let me know of some other place to publish this deployment instruction so that we can move on with the AMI.

**Comment by tomkraljevic** [comment]
i see.

then i would say just merge it, and improve iteratively once it’s live.

File: h2o-docs/src/product/productionizing.rst (hunk 1 above)
Comment (pratapram) [reply h2o-docs/src/product/productionizing.rst]: The inference server is not in the marketplace yet. They will not publish it unless they see this documentation.

**Comment by pratapram** [comment]
Please merge this whenever possible. AWS Marketplace team is waiting for this.

**Pull request merged**